src/chaddon.js

```diff
@@ -33,7 +34,35 @@ io.on('connection', function (socket) {
     socket.join(socket.channel);
 
     numUsers += 1;
-    addedUser = true;
+      addedUser = true;
+    io.to(socket.channel).emit('login', {
+      numUsers: numUsers
+    });
+
+    socket.broadcast.to(socket.channel).emit('user joined', {
```

Comment:
We should add a listener that handles an initial load. Similar to a controller that will set the appropriate state of the app.